### PR TITLE
[RFR] Fix rtd post split

### PR DESCRIPTION
**Pull request by SimonBiggs**
Woo hoo, got readthedocs to work:

https://readthedocs.org/projects/pymedphys/builds/8946448/

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -3,30 +3,11 @@
...
-    "editor.rulers": [
-      79,
-      119
-    ]
-  }
+  "editor.rulers": [
+    72,
+    79,
+    119
+  ]
```
Comment (Centrus007) [inline .vscode/settings.json]: Was this intentionally committed?

**Comment by SimonBiggs** [comment]
Ah yes, figured we could discuss it, but I think we can just have those
rulers everywhere.

And we I think letting VSCode automatically detect the indentation works
well enough...

**Comment by SimonBiggs** [comment]
Here's the link to the docs now building on rtd:

https://pymedphys.com/en/fix-rtd-post-split/

File: .vscode/settings.json (hunk 1 above)
Comment (SimonBiggs) [reply .vscode/settings.json]: Ah yes, figured we could discuss it, but I think we can just have those
rulers everywhere.

And we I think letting VSCode automatically detect the indentation works
well enough...

Comment (Centrus007) [reply .vscode/settings.json]: I personally found the 72 line a little annoying for `.md` and `.rst`...

Comment (Centrus007) [reply .vscode/settings.json]: Would something like:

```js
  "editor.rulers": [
    79,
    119
  ],
"[python]": {
    "editor.rulers": [
      72,
      79,
      119
    ]
  },
```

be alright?

**Comment by SimonBiggs** [comment]
Yup certainly

**Pull request merged**